eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -338,4 +338,10 @@
     <Field name="~(MAX_DURATION|NULL_TXN)"/>
     <Bug pattern="MS_SHOULD_BE_FINAL"/>
   </Match>
+
```

Comment:
[SpotBugs-P2]
This method uses the same code to implement two branches of a conditional branch. Check to ensure that this isn't a coding mistake.  
Already discussed with Service team, it is a breaking change, suggested to suppress it.